Makefile

```diff
@@ -5,6 +5,7 @@ BUILDTAGS=seccomp
 export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
 
 all:
+	ln -sfn $(CURDIR) $(CURDIR)/Godeps/_workspace/src/github.com/opencontainers/runc
 	go build -tags "$(BUILDTAGS)" -o runc .
```

Comment:
@hqhq I believe https://github.com/opencontainers/runc/pull/321 could be fixed so the tests would pass (https://github.com/gitido/runc/commit/d1659f75e252b503d505428829555314caac62a7), but both solutions are fine IMO :)